src/internal/m365/collection/exchange/collection.go

```diff
@@ -459,10 +481,13 @@ func (col *lazyFetchCollection) streamItems(
 			&lazyItemGetter{
 				userID:       user,
 				itemID:       id,
+				category:     col.FullPath().Category(),
 				getter:       col.getter,
 				modTime:      modTime,
 				immutableIDs: col.Opts().ToggleFeatures.ExchangeImmutableIDs,
 				parentPath:   parentPath,
+				skipChecker:  col.skipChecker,
+				opts:         col.BaseCollection.Opts(),
```

Comment:
could you please simplify this as well
```suggestion
				opts:         col.Opts(),
```